static/app/views/organizationStats/usageStatsPerMin.tsx

```diff
@@ -86,19 +76,17 @@ class UsageStatsPerMin extends DeprecatedAsyncComponent<Props, State> {
       dataCategory,
       getFormatUsageOptions(dataCategory)
     );
-  }
-
-  renderComponent() {
-    if (!this.minuteData) {
-      return null;
-    }
+  };
 
-    return (
-      <Wrapper>
-        {this.minuteData} {t('in last min')}
-      </Wrapper>
-    );
+  if (!minuteData()) {
+    return null;
   }
+
+  return (
+    <Wrapper>
+      {minuteData()} {t('in last min')}
+    </Wrapper>
+  );
```

Comment:
i think we should take this chance to fix the ordering of logic here.

pulling this out of the function and putting it higher so minuteData doesn't check them would make this component easier to read and remove the need to call minuteData twice
```
	if (isLoading || isError || !orgStats || orgStats.intervals.length === 0) {
      return undefined;
    }
```